c/doc/run_sample_on_intel_edison.md

```diff
@@ -54,7 +56,7 @@ You should see the following:
 
   ```
   $ opkg install git
-  $ git clone github.com/Azure/azure-iot-sdks.git
+  $ git clone --recursive https://github.com/Azure/azure-iot-suite-sdks.git
```

Comment:
This is cloning the iot suite? That was a private repo. Should be azure-iot-sdks.git.